packages/components/src/components/MultiDistributionChart/index.tsx

```diff
@@ -45,6 +49,19 @@ export const distributionSettingsSchema = yup.object({}).shape({
   showSummary: yup.boolean().required().default(false),
 });
 
+function scaleTypeToScale(
+  scaleType: ScaleType
+): d3.ScaleContinuousNumeric<number, number, never> {
+  switch (scaleType) {
+    case "linear":
+      return d3.scaleLinear();
+    case "log":
+      return d3.scaleLog();
+    case "exp":
+      return d3.scalePow().exponent(0.1);
```

Comment:
just fyi, I could imagine some situations later where we might want something more extreme than 0.1, like 0.01. Don't need to worry about now though.

Reply:
Yep, I'm not sure if it's worth it to support in the UI, but `Scale.power` takes an `exponent` parameter (and there's no default there).